src/oc/lib/api/common.clj

```diff
@@ -215,6 +247,11 @@
   :handle-unauthorized (fn [_] (unauthorized-response))
   :handle-forbidden  (fn [ctx] (if (:jwtoken ctx) (forbidden-response) (unauthorized-response)))})
 
+;; Same as anonymous-resource but also allows an identity token.
+(defn id-token-resource [passphrase]
+  (merge (anonymous-resource passphrase)
+         {:authorized? allow-id-token}))
+
```

Comment:
Hmmm i think this should be mostly like the authenticated-resouce, it should read the id token in initialize-context etc..

Reply:
I can document this function to indicate that it supports anonymous access but it says in the code `anonymous-resource`.